extensions-api/src/main/scala/pl/touk/nussknacker/engine/modelconfig/ModelConfigLoader.scala

```diff
@@ -100,8 +101,15 @@ abstract class ModelConfigLoader extends Serializable with LazyLogging {
       service1Url: ${baseUrl}/service1
       and have baseUrl taken from application config
      */
-    // We want to respect overrides (like system properties) and standard fallbacks (like reference.conf)
-    ConfigFactory.load(classLoader, withFallbackToDefaults(inputConfig, classLoader))
+    // We want to respect overrides (via environment variables) and standard fallbacks (like reference.conf).
+    // Code below works like 'ConfigFactory.load(ClassLoader,Config)' but it:
+    // - doesn't include system properties
+    // - always resolves overrides from environment variables (works as if 'config.override_with_env_vars' was always enabled')
+    ConfigFactory
+      .systemEnvironmentOverrides()
```

Comment:
IMO we should remove these overrides for the sake of clarity that it is done only on the designer stage but is not mandatory for now, feel free to omit this.

Reply:
Ok, I will remove them